packages/docusaurus-plugin-content-blog/index.js

```diff
@@ -25,8 +25,8 @@ const DEFAULT_OPTIONS = {
   routeBasePath: 'blog', // URL Route.
   include: ['*.md, *.mdx'], // Extensions to include.
   pageCount: 10, // How many entries per page.
-  blogPageComponent: path.resolve(__dirname, './components/BlogPage'),
-  blogPostComponent: path.resolve(__dirname, './components/BlogPost'),
+  blogPageComponent: '@theme/BlogPage',
```

Comment:
Mentioned this in discord. We need to remove aliases because what if some user install a “@theme/BlogPage” package from npm.

Reply:
Will be done as part of the UI framework stuff